src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const Navbar = () => {
+  return (
+    <div style={{border: "1px solid black"}}>
+      <h3>Navigation Bar Component</h3>
+      <span>Route Links  </span>
+      <Link to="/">/</Link>
+      <span>  ---  </span>
```

Comment:
The decorations should be handled with CSS. Looks like we can probably leverage a `ul` and change the style on that.

Reply:
This is temporary. I'll remove.